addon-test-support/-private/steps/then.js

```diff
@@ -119,19 +119,53 @@ const steps = {
         ? element.value
         : element.textContent.trim();
 
-    expect(regex.test(text)).true;
+    not
+      ? expect(regex.test(text)).false
+      : expect(regex.test(text)).true;
+  },
+
+  "Then $opinionatedElement should (not |NOT )?have the following text:\n$opinionatedText"() {
+    return steps["Then $opinionatedElement should (not |NOT )?(?:have text|say|be) \"(.*)\""](...arguments);
+  },
+
+  "Then $opinionatedElement should (not |NOT )?have the following text with collapsed whitespace: $opinionatedString"([collection/* , label, selector */], not, expectedText) {
```

Comment:
I wonder if this shouldn't be the default? As whitespace usually does not matter for HTML rendering (unless inside a `<pre>`). This step is now quite long to read. So maybe something like:
* `Then $opinionatedElement should (not |NOT )?have the following text: $opinionatedString"` -> collapse text before asserting
* `Then $opinionatedElement should (not |NOT )?have exactly the following text: $opinionatedString"` -> strict check

Reply:
I was also thinking about it but decided to stick to existing Chai-DOM and QUnit-DOM behavior.

Do you want me to change it now or just create an issue for future? Either way, the change should be considered breaking.